daprdocs/content/en/operations/monitoring/jaeger.md

```diff
@@ -0,0 +1,173 @@
+---
+type: docs
+title: "How-To: Set up Jaeger for distributed tracing"
+linkTitle: "Jaeger"
+weight: 3000
+description: "Set up Jaeger for distributed tracing"
+type: docs
+---
+Dapr currently supports two kind of tracing protocol: OpenCensus and
+Zipkin. Since Jaeger is compatible with Zipkin, we can use the Zipkin
+protocol to talk to Jaeger.
+
+## Configure self hosted mode
+
+### Setup
+
+For self hosted mode, on running `dapr init` the following YAML files are created by default and they are referenced by default on `dapr run` calls unless otherwise overridden.
+
+1. The following file in `$HOME/.dapr/components/zipkin.yaml` or `%USERPROFILE%\dapr\components\zipkin.yaml`:
```

Comment:
I suggest avoiding the requirement to change the default components files that are created on install. Those should probably stay as is. I think we should direct users to create YAML files and then use them in the run command with `--components-path'. That is much cleaner and allows a user to have no side effects on the default Dapr dev env

Reply:
That's a good idea. Fixed.